app/views/candidates/placement_requests/cancellations/_show_placement_request_cancelled_by_candidate.html.erb

```diff
@@ -33,9 +33,12 @@
       between 8:30am and 5:30pm, Monday to Friday.
     </p>
 
-    <h2 class="govuk-heading-m">
-      View and request more school experience
-    </h2>
-    <%= link_to 'Your requests and bookings', candidates_dashboard_path %>
+  <h2 class="govuk-heading-m">
+    Search for more school experience
+  </h2>
+    <p>
+      <%= govuk_link_to 'Search for school experience', new_candidates_school_search_path, class: "govuk-!-margin-bottom-0" %>
```

Comment:
@gemmadallmandfe this is what this looks like in this instance. It will be broadly similar across all of them.

<img width="468" alt="image" src="https://github.com/DFE-Digital/schools-experience/assets/35870975/deda882a-885e-4569-91c2-0599d2cd96a2">


This is how we make use of this 'search for school experience' button elsewhere, on the request confirmation page. We don't have it directly below a header, which might be a better approach.

<img width="481" alt="image" src="https://github.com/DFE-Digital/schools-experience/assets/35870975/81ee5f95-06e8-4ba0-93e0-bfc6a868161b">